npm/http/server/server.go

```diff
@@ -65,12 +65,14 @@ func NewNpmRestServer(listeningAddress string) *NPMRestServer {
 
 func (n *NPMRestServer) GetNpmMgr(npMgr *npm.NetworkPolicyManager) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		npMgr.Lock()
+		// QUESTION(jungukcho) What information does it need from npMgr?
+		// Is this function required to hold lock for npMgr?
+		// npMgr.Lock()
```

Comment:
This function is used by Dataplane parser Quang uses, to get the whole internal cache for PodMap, Nsmap and netpolmap, since this is a read operation, i will let your design drive if lock is required.

But please do make sure that this function will return the 3 maps along with listmap and setmap